### DISCUSSION TITLE
Darken text in caption dialog?

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1319] [Support] [answered]
Can we darken the text in the caption dialog? 

It disappears on me: 

[Screenshot](https://i.imgur.com/yTfuWfZ.png)

**Comment by idreyn** [comment, marked as answer]
The rich text toolbar in the Page block editor is on its last legs and we're currently in the process of replacing it with the same one that exists on Pubs:

![image](https://user-images.githubusercontent.com/2208769/112173747-3abd3600-8bcc-11eb-81c2-217459f21bde.png)

 Look for this change to land sometime next week!

**Comment by writingpoli** [comment]
Yay!

Sent from ProtonMail mobile

-------- Original Message --------
